doc/releases/release-notes-3.3.rst

```diff
@@ -630,6 +630,8 @@ Drivers and Sensors
 
 * Display
 
+  * No major changes except for minor bug fixes.
```

Comment:
If there is nothing noteworthy, the display section should be left empty (I will remove any empty sections later).

Reply:
okay,  I was just confused about the explicit request to add it.